pom.xml

```diff
@@ -212,6 +212,11 @@
     <dependencies>
         <!-- This explicit dependency is added because the older version
             which gets added by parent pom has the vulnerability -->
+        <dependency>
+            <groupId>org.jenkins-ci.modules</groupId>
```

Comment:
Did we check that this dependency is not transitive to the parent pom of org.jenkins-ci.plugins? However, I doubt it as the group id is different.

Reply:
Instance identity is not available if it's not specified as a dependency